sql/testdata/subquery

```diff
@@ -61,3 +61,39 @@ query I
 SELECT (SELECT a FROM abc WHERE false)
 ----
 NULL
+
+query I
+VALUES (1, (SELECT (2)))
+----
+1 2
+
+statement ok
+INSERT INTO abc VALUES ((SELECT 7), (SELECT 8), (SELECT 9))
```

Comment:
Yes. And there are corresponding cases for `UPDATE`. These found bugs in insert/update which are now fixed.